src/auth/authController.js

```diff
@@ -43,4 +44,22 @@ const signup = async (req, res, next) => {
   }
 };
 
-export default { signup };
+const login = async (req, res, next) => {
+  const { email } = req.body;
+  try {
+    const admin = await Person.findOne({ where: { email: email } });
+    const token = jwt.sign(
+      { email: admin.email, adminId: admin.id },
```

Comment:
- I would make here `person.id` since tomorrow, all the users will use this route, not just admin's.